src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -85,3 +85,13 @@ def validate_app_exists_in_rg(cmd, namespace):
             app = client.web_apps.get(resource_group_name, webapp, None, raw=True)
         if app.response.status_code != 200:
             raise CLIError(app.response.text)
+
+
+def validate_app_exists_in_rg(cmd, namespace):
+    client = web_client_factory(cmd.cli_ctx)
+    if isinstance(namespace.name, str) and isinstance(namespace.resource_group_name, str):
```

Comment:
This is unnecessary, as the command line parser should have taken care of this already. I know you copied and pasted from the old code, but I don't think you need it here, and probably it is unnecessary in the old place either

Reply:
makes sense - yes, i did this while emulating the older existing code. Will remove these - will probably need to update the tests as well.